.github/workflows/release.yml

```diff
@@ -30,10 +30,10 @@ jobs:
         run: |
           if [ -n "${{ github.event.head_commit.message }}" ]
           then
-            echo "commit-message=${{ github.event.head_commit.message }}" >> "$GITHUB_OUTPUT"
+            printf "commit-message=%q\n" "${{ github.event.head_commit.message }}" >> "$GITHUB_OUTPUT"
```

Comment:
This doesn't work for me in non-posix system (tested in docker). I think we could use `echo -e` or pipe to `head -n 1`